test/resources/examples/todo/auth-middleware.hbs

```diff
@@ -0,0 +1 @@
+,
```

Comment:
There is something wrong with the //#Filename parser since this should not be created

Reply:
Stray comma did that